### ISSUE TITLE
docker.tag.long usage

### ISSUE THREAD

**Issue by jmcc0nn3ll** [GoogleCloudPlatform/jetty-runtime#123]
Currently the jetty9 module tags with a long timestamp format and this pollutes a local docker repository.

We use it for test behavior though so we still need it, but it would be good to perhaps clean this up after running tests.

Is there a reason to have tag.short and tag.long outside of the test usage and does smoking the tag.long at the end of the build raise any red flags?

**Comment by meltsufin** [comment]
I do agree that we should probably provide the ability to clean up the created images in the local repo, but I don't think it's sufficient to remove the long tag because the image will still remain and keep taking up space. So, we would want to force delete the whole image with all of its tags.

I think the more appropriate place for this cleanup is to bind a goal to the "clean" phase, but then there's the challenge of figuring out which images were created by the project.

**Comment by jmcc0nn3ll** [comment]
well, can just have an execution bound to clean that operates a bit like your bash magic one piped through xargs to remove based on a tag expression

Isuppose we could tag for testing specifically and then just clean those tags, ex. {docker.tag.short}-test.

**Comment by meltsufin** [comment]
Yes, I guess we can just add an additional tag like `jetty:{docker.tag.long}-test`, and then delete all of the images that match that pattern in clean phase.
